3702/0/about_control_statements.rb

```diff
@@ -0,0 +1,135 @@
+# rubocop:disable Metrics/ClassLength, Lint/LiteralAsCondition, Metrics/MethodLength
+
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+class AboutControlStatements < Neo::Koan
+  def test_if_then_else_statements
+    result = if true
+               :true_value
+             else
+               :false_value
+             end
+    assert_equal :true_value, result
+  end
+
+  def test_if_then_statements
+    result = :default_value
+    result = :true_value if true
+    assert_equal :true_value, result
+  end
+
+  def test_if_statements_return_values
+    value = if true
+              :true_value
+            else
+              :false_value
+            end
+    assert_equal :true_value, value
+
+    value = if false
```

Comment:
The same as above

Reply:
... also all 3 last issues might be changed into one-liners using ternary operator